accname/name/comp_host_language_label.tentative.html

```diff
@@ -0,0 +1,118 @@
+    <h2>HTML input with role="none" and disabled with label associated</h2>
+    <h3>HTML input label/for</h3>
+    <label for="irpd">input label</label>
+    <input id="irpd" type="text" data-expectedlabel="" data-testname="html: label[for]+input[role=none][disabled] (role none applied due to disabled input)" class="ex-label" role="none" disabled>
+    <h3>HTML label encapsulation</h3>
+    <label>
+      <span>input label</span>
+      <input type="text" data-expectedlabel="" data-testname="html: label>input[role=none][disabled] (with label encapsulation, role none applied due to disabled input)" class="ex-label" role="none" disabled>
+    </label>
+
+    <h2>HTML img with role="none" and non empty alt</h2>
+    <img src="https://www.w3.org/assets/logos/w3c/w3c-no-bars.svg" role="none" alt="w3c logo" data-expectedlabel="" data-testname="html: img[role=none][alt]:not([alt='']) (role none takes precedence over empty alt)" class="ex-label">
+
+    <h2>SVG wrapping title with role="none"</h2>
+    <svg viewBox="0 0 20 10" xmlns="http://www.w3.org/2000/svg">
+      <circle cx="5" cy="5" r="4" data-expectedlabel="circle" data-testname="svg: svg>circle>title[role=none] (role none set on title doesn't affect circle acc name)" class="ex-label">
+        <title role="none">circle</title>
+      </circle>
+    </svg>
+
+    <h2>HTML fieldset wrapping legend with role="none"</h2>
+    <fieldset data-expectedlabel="legend" data-testname="html: fieldset>legend[role=none] (role none set on legend doesn't affect fieldset acc name)" class="ex-label"><legend role="none">legend</legend></fieldset>
+
+    <h2>HTML fieldset with role="none" wrapping legend</h2>
+    <fieldset role="none" data-expectedlabel="" data-testname="html: fieldset[role=none]>legend (role none set on fieldset affects fieldset acc name)" class="ex-label"><legend>legend</legend></fieldset>
+
+    <h2>HTML fieldset role="none" wrapping legend with role="none"</h2>
+    <fieldset role="none" data-expectedlabel="" data-testname="html: fieldset[role=none]>legend[role=none] (role none set on fieldset affects fieldset acc name)" class="ex-label"><legend role="none">legend</legend></fieldset>
+
+    <h2>HTML table wrapping caption with role="none"</h2>
+    <table data-expectedlabel="caption" data-testname="html: table>caption[role=none] (role none set on caption doesn't affect table acc name)" class="ex-label">
+      <caption role="none">caption</caption>
+      <tr>
+        <th>th1</th>
+        <th>th2</th>
+      </tr>
+      <tr>
+        <td>td1</td>
+        <td>td2</td>
+      </tr>
+    </table>
+
+    <h2>HTML table with role="none" wrapping caption</h2>
+    <table role="none" data-expectedlabel="" data-testname="html: table[role=none]>caption (role none set on table affects fieldset acc name)" class="ex-label">
```

Comment:
This is one of the tests that should be moved to a new "blocked by testability" issue. You can't validly (yet) expect to be able to test computedlabel on an element with no role... No engines compute this... I think Chrome just returns the empty string when there is no backing object, but that's an implementation detail, not a test.